.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -31,5 +31,5 @@
 - [ ] Attached Screenshots, Payload Example
 - [ ] Unit, Integration Tests
 - [ ] Documentation Updates (when adding/changing features)
-- [ ] Example Plugin Updates (when adding/changing features)
+- [ ] Example App Updates (when adding/changing features)
```

Comment:
This shouldn't be in the app cookiecutter at all - this is specific to Nautobot core?

Reply:
Thanks, I will fix this in the template, and push it with a new re-bake.